File: HW2/mrzeeifelse.f90
program mrzee
implicit none
integer :: temperature, month, coin

print *, "It is time for Mr. Zee to partake in some leisure activity. "
print *, "What is the temperature?"
read *, temperature

if (temperature >=72 .and. temperature <=80) then
  print *, "Mr. Zee will go on a long walk."
  print *, "What is the number of the current month?"
  read *, month
  if (month >6 .and. month<10) then
    print *, "Mr. Zee will bring his umbrella."
  else
    print *, "Mr. Zee will not bring his umbrella."
  endif
else
  print *, "Mr. Zee will stay in and read."
  print *, "Flip a coin. Input 1 if heads, anything else if tails."
  read *, coin
  if(coin == 1) then
    print *, "Mr. Zee will read science fiction."
  else
    print *, "Mr. Zee will read Greek mythology."
  endif
endif

end program mrzee

File: HW2/mrzeeifelse.c
#include <stdio.h>

int main(){
  int temperature, month, coin;
  printf("It is time for Mr. Zee to partake in some leisure activity.\n");
  printf("What is the current temperature in °F?\n");
  fflush(stdout);
  scanf("%d",&temperature);
  if (temperature <= 80 && temperature >= 72){
    printf("Mr. Zee is going on a long walk.\n");
    printf("What is the number of the current month?\n");
    fflush(stdout);
    scanf("%d",&month);
    if(month > 7 && month < 11)
      printf("Mr. Zee will bring his umbrella.\n");
    else
      printf("Mr. Zee will not bring his umbrella.\n");
  }
  else{
    printf("Mr. Zee will stay home and read.\n");
    printf("Flip a coin. If heads, input 1. If tails, input anything else.\n");
    fflush(stdout);
    scanf("%d",&coin);
    if(coin == 1)
      printf("Mr. Zee will read science fiction.\n");
    else
      printf("Mr. Zee will read Greek mythology.\n");
}}
